pkg/controller/servicebindingrequest/servicebinder_test.go

```diff
@@ -573,3 +572,27 @@ func TestServiceBinder_Bind(t *testing.T) {
 		},
 	}))
 }
+
+func TestEnsureDefaults(t *testing.T) {
```

Comment:
It is missing a test case where the containers path has been specified and `ensureDefaults` does not overwrite its results.

Reply:
Done. 5c6c706f0cb817bc42e6864557e916eb9a43bf2f